cmd/hub/mcscontroller/controller.go

```diff
@@ -83,8 +86,12 @@ func (r *ServiceExportReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 			return ctrl.Result{}, err
 		}
 	}
+	listOptions, err := lhutil.ServiceExportListFilter(se.ObjectMeta)
+	if err != nil {
+		return ctrl.Result{}, err
+	}
 
-	result, err := r.reconcile(ctx, lhutil.GetOriginalObjectName(se.ObjectMeta))
+	result, err := r.reconcile(ctx, lhutil.GetOriginalObjectName(se.ObjectMeta), listOptions)
```

Comment:
`lhutil.ServiceExportListFilter` needs the `objmd metav1.ObjectMeta` of the object (which reconcile doesn't have),
I thought its the outer function (Reconcile) responsibility to get the listOptions, which keeps reconcile function "unaware" of the object it working on.

Reply:
I dont think its a good idea. to add this param. cant list options can be created inside reconcile() from the name and namespace ?

Reply:
In order to create the list options we need the `ObjectMeta`, which we don't have access to. 
I can refactor the `ServiceExportListFilter` function to receive the name and namespace instead and than it would be possible

Reply:
I think you should..